css/nav.css

```diff
@@ -13,7 +13,6 @@ nav * {
     color: black !important;
     padding: 0px !important;
     margin-top: 16px !important;
```

Comment:
Išmetam !important, to jei norimas stilius nesuveikia:
- perdėlioti CSS failų eiliškumą, jog komponentai būtų žemiau bendrinio stiliaus;
- arba padaryti didesnio specifiškumo selector'ių.